sql/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -215,15 +215,46 @@ message Expression {
     }
 
     message Array {
```

Comment:
I personally feel it might be better to introduce new messages for this purpose, so that we can minimize the code changes, and the conversion logic in the server side can be more clear